easy/admin/field.py

```diff
@@ -85,6 +88,39 @@ def render(self, obj):
         return self.default
 
 
+class ManyToManyAdminField(SimpleAdminField):
```

Comment:
:+1:

Reply:
I will improve the coverage by the way. I put it together after using it myself, but I haven't had time to improve the testing.

Reply:
Great